pkg/apis/v1alpha1/stage_types.go

```diff
@@ -63,19 +63,19 @@ type StageResourceRef struct {
 
 // StageDelay describes the delay time before going to next.
 type StageDelay struct {
-	// Duration indicates the stage delay time.
-	// If JitterDuration is less than Duration, then JitterDuration is used.
-	Duration metav1.Duration `json:"duration"`
+	// MilliSeconds indicates the stage delay time.
```

Comment:
I think it is more appropriate to use `DurationMilliSeconds` and `JitterDurationMilliSeconds` as names, and `MilliSeconds` alone is not appropriate.